scss/modules/common/_extends.scss

```diff
@@ -0,0 +1,6 @@
+%image_inner {
```

Comment:
rename it like `image-holder` or `image-container`

Reply:
In my view, your names are't correct. It's not container, It's image inside container. But I also couldn't think up with a better name))) Offer please something else))